test/components/tableTests.ts

```diff
@@ -471,6 +471,38 @@ describe("Tables", () => {
         table.destroy();
         svg.remove();
       });
+
+      it("rowPadding and columnPadding error on invalid input", () => {
+        const svg = TestMethods.generateSVG();
+        const component0 = new Plottable.Component();
+        const component1 = new Plottable.Component();
+        const component2 = new Plottable.Component();
+        const component3 = new Plottable.Component();
```

Comment:
variables are not used other than in table declaration, maybe just create components inline?